src/SFA.DAS.ApprenticeCommitments.Jobs/NewApprenticeship.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace SFA.DAS.ApprenticeCommitments.Jobs
+{
+    public class NewApprenticeship
```

Comment:
New is a bit of reserved word so I always hate it. Should this class be called CreateApprenticeshipRequest

Reply:
That depends on if we're trying to be "rest-ish".  If so, then we should be posting a resource to the API, rather than calling a function.  In this case, the resource I was thinking of was a "new apprenticeship" resource - as that is what has been created.  

I think this is likely to change somewhat as we evolve the project.  For now I'm happy to rename it to simply `Apprenticeship`?

Maybe discuss tomorrow.